src/inmanta/export.py

```diff
@@ -441,7 +441,7 @@ def commit_resources(self, version: int, resources: List[Dict[str, str]], metada
         tid = cfg_env.get()
         if tid is None:
             LOGGER.error("The environment for this model should be set!")
-            return
+            raise Exception("The environment for this model should be set!")
```

Comment:
This is not exactly the most specific exception type, but it is in line with the rest of the method